crc-support/oci/lib/unix/run.sh

```
@@ -104,17 +122,24 @@ if [[ $download == "true" ]]; then
     # Check if require download
     required_download $aName $aSHAName
     if [[ ${?} -ne 0 ]]; then
-        # Required to download
-        rm -f $aName
-        dURL="$aBaseURL/$aName"
-        download $dURL
-        check_download $aName $aSHAName
-        if [[ ${?} -ne 0 ]]; then 
+        local doExit=0
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)